.gitignore

```
@@ -0,0 +1 @@
+.nextflow*
```

`work` is another directory produced by nextflow that doesn't need to be source controlled: https://www.nextflow.io/docs/latest/cli.html#clean

```suggestion
.nextflow*
work
```